### ISSUE TITLE
Chart data is lost when leaving Panels tab and returning

### ISSUE THREAD

**Issue by davidgilbertson** [comet-ml/issue-tracking#492]
## Describe the Bug
Edit a panel chart so that the x-axis min is `0`. Do not set a max. Save it.

![image](https://user-images.githubusercontent.com/4443482/209884168-1be280d0-1348-477d-aa4e-0afa929aa5eb.png)

Now, whenever you leave the Panels tab and return, the chart x-axis range is set to `(-1, 6)` and you have to select 'reset zoom' to see the chart with the correct axis range.

https://user-images.githubusercontent.com/4443482/209883890-705406b3-7fc5-4978-a29f-6a5de6e7b974.mp4

## Expected behavior
I should be able to set just the min. I assume this is very common as the first step when I log my training loss is usually sooner than the step where I log my validation accuracy. But I would like both charts to align (visually, when vertically stacked in the UI), and thus both start at 0. But I don't want to set a maximum, because I don't know in advance what that will be.

And indeed this seems possible, but breaks when returning to the page making it very frustrating to use.

I assume you have some code somewhere that assumes that if `min` is set, `max` is too, and has a fallback for when both aren't set. A quick fix would be to call 'reset zoom' when first loading the chart as that's implemented correctly.

## Where is the issue?
- [ ] Comet Python SDK
- [x] Comet UI
- [ ] Third Party Integrations (Huggingface, TensorboardX, Pytorch Lighting etc)
